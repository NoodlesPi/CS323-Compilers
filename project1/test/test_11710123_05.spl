int zhekehaonana(
{
  int a;
  a = 3a;
}