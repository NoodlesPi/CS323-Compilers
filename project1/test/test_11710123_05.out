Error type B at Line 1: Missing closing parenthesis ')'
Error type A at Line 4: unknown lexeme 3a
